src/sandstorm/web-session.capnp

```diff
@@ -225,9 +313,16 @@ interface WebSession @0xa50711a14d35a8ce extends(Grain.UiSession) {
       noContent :group {
         # Return successful, but with no content (status codes 204 and 205)
 
+        statusCode @17 :SuccessCode;
```

Comment:
Sharing this enum with `content` creates ambiguity: What happens if I send `noContent` with `statusCode = ok`?

It looks to me like this isn't needed, particularly if `notModified` becomes a completely different member of the union. There are some places in your PR that suggest that "207 Multi Status" can come in both `content` and `noContent` varieties, but looking at the WebDAV standard I don't see this -- it looks to me like it should always have content, right? I think it belongs under `content`, and if the body is empty, then the body is empty. `noContent` was meant specifically for status codes where having an entity body would be illegal.